cc/main.cc

```diff
@@ -102,8 +95,39 @@ DEFINE_double(
 DEFINE_double(decay_factor, 0.98,
               "If time_limit is non-zero, the decay factor used to shorten the "
               "amount of time spent thinking as the game progresses.");
+DEFINE_bool(run_forever, false,
+            "When running 'selfplay' mode, whether to run forever.");
 
-DEFINE_string(mode, "", "Mode to run in: \"selfplay\", \"eval\" or \"gtp\"");
+// Inference flags.
+DEFINE_string(model, "",
+              "Path to a minigo model. If remote_inference=false, the model "
+              "should be a serialized GraphDef proto. If "
+              "remote_inference=true, the model should be saved checkpoint.");
+DEFINE_string(model_two, "",
+              "When running 'eval' mode, provide a path to a second minigo "
+              "model, also serialized as a GraphDef proto.");
+DEFINE_bool(remote_inference, false,
+            "If true, run the model using the InferenceServer. This launches "
+            "a Python subprocess that performs the actual inference. Required "
+            "when running on Cloud TPU.");
+DEFINE_int32(port, 50051,
+             "The port opened by the InferenceService server.");
+DEFINE_string(tpu_name, "", "Cloud TPU name, e.g. grpc://10.240.2.2:8470.");
+DEFINE_int32(parallel_games, 32, "Number of games to play in parallel.");
```

Comment:
is there a comment somewhere that parallel_games should equal games_per_inference * 2 for perf?

Reply:
done.